CONTRIBUTING.md

```diff
@@ -4,33 +4,35 @@ Contributions are welcome, and they are greatly appreciated! Every little bit
 helps, and credit will always be given.
 
 ## Get Started!
+
 Ready to contribute? Here's how to set up `aicsimageio` for local development.
 
-1. Fork the `aicsimageio` repo on GitHub.
+1.  Fork the `aicsimageio` repo on GitHub.
```

Comment:
noise?

Reply:
VSCode autoformatting. I liked it enough to keep.